### PR TITLE
add download link in error page

### PR DESCRIPTION
**Pull request by kangjae4real**
Solved #1852 

add nine-chronicle download link in "/error/reinstall" page

### CODE REVIEW

**Comment by Atralupus** [comment]
Thanks for your first contributing

File: src/v2/views/ErrorView/index.tsx [hunk 1]
```diff
@@ -133,8 +135,14 @@ function ErrorView() {
         </ErrorContent>
       </Route>
       <Route path={`${path}/reinstall`}>
-        <ErrorContent title={t("Reinstall required", { _tags: transifexTags })}>
+        <ErrorContent
+          title={t("Reinstall required edit edit", { _tags: transifexTags })}
+        >
           <T _str="Please reinstall Nine Chronicles." _tags={transifexTags} />
+          <T
```
Comment (BasixKOR) [inline src/v2/views/ErrorView/index.tsx:142]: You can navigate the user to the browser via: https://www.electronjs.org/docs/latest/api/shell#shellopenexternalurl-options

**Review by BasixKOR — COMMENTED** [review]
Thanks for your contribution!

File: src/v2/views/ErrorView/index.tsx [hunk 2]
```diff
@@ -12,6 +12,8 @@ import bytes from "bytes";
 import { getBlockChainStorePath, userConfigStore, app } from "src/config";
 
 const transifexTags = "v2/ErrorView";
+const downloadLink =
+  "https://release.nine-chronicles.com/NineChroniclesInstaller.exe";
```
Comment (Atralupus) [inline src/v2/views/ErrorView/index.tsx:16]: You can use `baseUrl` instead `https://release.nine-chronicles.com`
https://github.com/planetarium/9c-launcher/blob/cdf051eca964581c9e034fa65e33df559443e5b3/src/config.ts#L224 

and recommend define `var installerName = "NineChroniclesInstaller.exe"` in config.ts

File: src/v2/views/ErrorView/index.tsx (hunk 1 above)
Comment (kangjae4real) [reply src/v2/views/ErrorView/index.tsx:142]: Sorry for i late
I'll try the way you told me.
thx!! 🤟🏻

File: src/v2/views/ErrorView/index.tsx (hunk 2 above)
Comment (kangjae4real) [reply src/v2/views/ErrorView/index.tsx:16]: Oh thx! 👍🏻 
I fix it

**Comment by kangjae4real** [comment]
스크린샷 2022-10-10 오후 7 36 32

In my personal opinion, I thought it was right to add a Checkbox and give the user the option to reinstall.

When the user checks the Checkbox, the browser is opened when the user clicks the "Restart" Button and downloads the installer.

In the opposite situation, the Client simply restarts.

I'd appreciate it if you could review it. 🙇🏻 🙇🏻

File: src/v2/views/ErrorView/index.tsx [hunk 3]
```diff
@@ -31,6 +37,15 @@ function handleRestart() {
 function ErrorView() {
   const { path } = useRouteMatch();
   const [state] = useActor(preloadService);
+  const checkboxRef = useRef<HTMLInputElement>(null);
+
+  const onRestartButtonClick = useCallback(() => {
```
Comment (Atralupus) [inline src/v2/views/ErrorView/index.tsx:42]:
```suggestion
  const handleRestartClick = useCallback(() => {
```
Isn't 'handle' correct? cc @BasixKOR

Comment (BasixKOR) [reply src/v2/views/ErrorView/index.tsx:42]: It's up to the taste, but I'd prefer `handle` too.

**Review by BasixKOR — APPROVED** [review]

**Review by Atralupus — APPROVED** [review]
Gooood

**Pull request merged**